### ISSUE TITLE
Problem with proxy middleware

### ISSUE THREAD

**Issue by vinhtq** [MillerRen/vue-boilerplate#2]
I always have this error when run ```npm run dev```

```
[HPM] Proxy created: /api  ->  http://localhost:9000
[HPM] Subscribed to http-proxy events:  [ 'error', 'close' ]
```

So that you cannot make any request to server route

**Comment by MillerRen** [comment]
Have you started the server? You have to run it by `node server app.js` or provide another server. I will improve readme.

**Comment by vinhtq** [comment]
Yes I just figure how to start another server process. But the server needs some libs which are not exist in package.json such as: mongoose, cors... 

-------- Original message --------

**Comment by MillerRen** [comment]
I moved server packages to `server` directory. You need to install them.
